package exceptionhandling;

public class ExceptionExample {
public static void main(String[] args) {
	try {
		int result=100/0;
		System.out.println("Result :"+result);
	} catch (ArithmeticException e) {
		System.out.println(e.getMessage());
	}
	System.out.println("Program continues");
}
}
